spyder/config/main.py

```diff
@@ -438,6 +438,8 @@
               'editor/debug cell': 'Alt+Shift+Return',
               'editor/go to next cell': 'Ctrl+Down',
               'editor/go to previous cell': 'Ctrl+Up',
+              'editor/scroll line down': 'Ctrl+Shift+Down',
+              'editor/scroll line up': 'Ctrl+Shift+Up',

```

Comment:
I do not think these shortcuts make a lot of sense, on mac.

I would expect that to select the whole document on one direction or the other I think these shortcuts should be empty and let the user use them as they see fit

Reply:
That's something to be solved with a new set of shortcuts specific for macOS, as you mentioned in another issue, not here, right?

Reply:
I think you are both right, but for the moment, the solution proposed by @goanpeca to just let these shortcuts empty and let the user use them as they see fit is the best.

Reply:
>That's something to be solved with a new set of shortcuts specific for macOS, as you mentioned in another issue, not here, right?

Yes but something to consider is that there will always be less option for shortcuts on Mac since a lot of keyboards do not have a PGUp/PgDown and that is usually what Cmd+Up/Cmd+Down  translates to.

There is also User Interface Guidelines which do not really mention this at all. I am not sure what the current behavior is if a shortcut is `Empty` but I do not think we have to always define shortcut values.

Reply:
> I am not sure what the current behavior is if a shortcut is Empty but I do not think we have to always define shortcut values.

The shortcut is simply not bind to any key sequence, so that is not a problem.

Reply:
@ccordoba12 @goanpeca Just to be sure, do I go with the option where I leave this new shortcut blank?

Reply:
Yes, please do.